krabs/krabs/filtering/comparers.hpp

```diff
@@ -8,9 +8,9 @@
 #pragma warning(disable: 4800) // bool warning in boost when _MANAGED flag is set
 #endif
 
+#include <algorithm>
+
 #include "../compiler_check.hpp"
-#include <boost/algorithm/string.hpp>
```

Comment:
Because you've got rid of boost, you can also get rid of the pragma warning stuff above and below.

Reply:
Good point, thanks!